autoload/tagbar.vim

```diff
@@ -1158,6 +1158,20 @@ function! s:ProcessFile(fname, ftype) abort
         return
     endif
 
+    " If the file size limit it set, then check the linecount to see if
```

Comment:
I like this feature, but in reviewing the code I found it very counter intuitive that the thing being checked is lines, but the setting here is called filesize. File size in my mind is a byte length, not a line count.

One of the most common cases I'd like to exclude is "compiled" source files that have been compressed by removing spaces. These show up a lot in JavaScript work and are usually massive rollups of other code jammed into just a few lines. They would easily be over a reasonable limit for filesize, but not trip a line count limit.

I think either this setting should be renamed to be intuitive or changed so it actually works on filesize. I think I'd prefer the latter. What do you think?

Reply:
Updated to work on file byte size instead.